.github/workflows/fusequery-unit-tests.yml

```diff
@@ -5,6 +5,20 @@ env:
   CARGO_TERM_COLOR: always
 
 jobs:
+  miri-checks:
+    name: Miri
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+      - name: Install Rust
+        run: |
+          rustup toolchain install nightly-2021-01-19
```

Comment:
It wouble be better to install miri here:
https://github.com/datafuselabs/fuse-query/blob/master/scripts/dev_setup.sh#L157